packages/config/lib/store.js

```diff
@@ -77,19 +81,37 @@ class Store {
       }
     }
 
-    if (app === undefined) {
-      const attemptedToRunVersion = this.getVersionFromSchema($schema)
+    const match = $schema?.match(/\/schemas\/(.*)\/(.*)/)
+    if (!app && match) {
+      const type = match[2]
+
+      const toLoad = `https://platformatic.dev/schemas/v${pltVersion}/${type}`
+      app = this.#map.get(toLoad)
+    }
 
-      if (attemptedToRunVersion === null) {
-        throw new errors.AddAModulePropertyToTheConfigOrAddAKnownSchemaError()
+    // Legacy Platformatic apps
+    if (!app && $schema?.indexOf('./') === 0) {
+      if (core || db) {
+        app = this.#map.get(`https://platformatic.dev/schemas/v${pltVersion}/db`)
       } else {
-        throw new errors.VersionMismatchError(this.#currentVersion, attemptedToRunVersion)
+        app = this.#map.get(`https://platformatic.dev/schemas/v${pltVersion}/service`)
```

Comment:
It looks like it might break the case when you use local config schema with your custom stackable. If we still support this usecase.